### ISSUE TITLE
Anyway to use the -r flag without blanking-out the background ?

### ISSUE THREAD

**Issue by jamesee** [popcornmix/omxplayer#687]
@popcornmix  
Hi, I am facing audio and video out of sync issue. If tried the -r flag (which adjust the framerate to video) and the omxplayer play the TV-stream perfectly. Unfortunately, the -r flag blank-out the background.

omxplayer --live --win 0,434,1080,1042 -r  $CH_URL 

My question: 
Is there anyway to use the -r flag WITHOUT blank-out the background?

**Comment by popcornmix** [comment]
No, there is no way of changing hdmi mode without destroying any overlay layers (including framebuffer).
If you normally play content with the same framerate then using that hdmi as a default (hdmi_group/hdmi_mode in config.txt) may be a workaround.
